### PR TITLE
make status nullable

### PR DESCRIPTION
**Pull request by smoqadam**

### CODE REVIEW

File: src/ViewModel/ContentAside.php [hunk 1]
```diff
@@ -18,7 +18,7 @@ final class ContentAside implements ViewModel
     private $related;
 
     public function __construct(
-        ContentAsideStatus $status,
+        ContentAsideStatus $status = null,
```
Comment (nlisgo) [inline src/ViewModel/ContentAside.php:21]: Please can we add a test.

**Review by nlisgo — APPROVED** [review]
test needed

File: src/ViewModel/ContentAside.php (hunk 1 above)
Comment (smoqadam) [reply src/ViewModel/ContentAside.php:21]: @nlisgo Tests added

File: tests/src/ViewModel/ContentAsideTest.php [hunk 2]
```diff
@@ -99,11 +99,13 @@ public function it_has_data()
...
-    public function it_must_have_a_title()
+    public function it_may_have_a_status()
     {
-        $this->expectException(InvalidArgumentException::class);
+        $contentAside = new ContentAside(new ContentAsideStatus("content aside"));
+        $this->assertSame('content aside', $contentAside['status']['title']);
 
-        new ContentAside(new ContentAsideStatus(''));
+        $contentAside = new ContentAside();
+        $this->assertNull($contentAside['status']);
```
Comment (nlisgo) [inline tests/src/ViewModel/ContentAsideTest.php:108]: We have used the variables `$with` and `$without` fairly consistently throughout the tests in this library. 

See: https://github.com/elifesciences/patterns-php/blob/0b2f593568ec06abc4958cef10fdddad57f2ff68/tests/src/ViewModel/AboutProfileTest.php#L73-L82

**Review by nlisgo — CHANGES_REQUESTED** [review]

**Review by nlisgo — APPROVED** [review]

**Pull request merged**